leetcode/linked-list.md

```diff
@@ -0,0 +1,235 @@
+# linked-list
```

Comment:
ALL WRONG in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message